src/components/Table/Table.tsx

```diff
@@ -0,0 +1,257 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+
+import { TABLE } from '../ThemeIdentifiers';
+
+import TableHeader from './TableHeader';
+import TableHead from './TableHead';
+import TableBody from './TableBody';
+import TableRow from './TableRow';
+import TableData from './TableData';
+
+import * as baseTheme from './Table.scss';
+
+export interface Props {
+  // To make table bordered
+  bordered?: boolean;
+  // Column config, which renders the header
+  column?: any;
```

Comment:
@dheeraja00 We need to explicity define an interface for this. Check this out: https://www.typescriptlang.org/docs/handbook/interfaces.html